src/main/views/user-option.njk

```diff
@@ -28,7 +30,7 @@
           hint: {
             text: "Please select an option to continue"
           },
-          errorMessage: { text: errorMessage } if hasError,
+          errorMessage: { text: error.userAction.message } if error.userAction,
```

Comment:
What is the benefit of using error,userAction.message and error.email.message rather than just error.message?

Reply:
Because 'error.message' is ambiguous, especially on pages where multiple errors could occur (not necessarily this page but keeping it consistent). 

Employing the fieldId as part of the accessor allows us to easily list multiple errors on a page without ambiguity as to what each message text relates to.

Reply:
I guess that is for future proofing. Couldn't find examples like that in the code but guess we might have that case later :)